# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../freeglut/missing_lstdc++.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./configure.ac.orig	2006-10-24 14:45:07.000000000 +0300
+++ ./configure.ac	2006-10-24 14:44:41.000000000 +0300
@@ -26,7 +26,7 @@
   VERSION_INFO=
   EXPORT_FLAGS="-DFREEGLUT_EXPORTS"
 else
-  GL_LIBS="-lGL -lGLU -lXext -lX11"
+  GL_LIBS="-lGL -lGLU -lstdc++ -lXext -lX11"
   AC_CHECK_LIB([Xxf86vm], [XF86VidModeSwitchToMode],
                [LIBXXF86VM=-lXxf86vm], [LIBXXF86VM=],
                [$X_LIBS -lXext -lX11])
